lib/containers/container_images.pm

```diff
@@ -78,7 +51,7 @@ sub build_and_run_image {
     die "You must define the runtime!"    unless $runtime;
     die "You must define the Dockerfile!" unless $dockerfile;
 
-    my $dir = "/root/containerapp";
+    my $dir = "~/containerapp";
```

Comment:
why? Even if we run with root, wouldn't it be safer to keep full path?

Reply:
We run podman also under normal user ;)

Reply:
Ah, in rootles mode, ok.